server/repos/campgrounds.ts

```diff
@@ -0,0 +1,78 @@
+import mongoose, { Schema } from "mongoose";
+import { faker } from "@faker-js/faker";
+import cities from "../../src/seeds/cities.ts";
+
+const opts = { toJSON: { virtuals: true } };
+
+const campgroundSchema = new Schema(
+  {
+    title: String,
+    images: [{ url: String, filename: String }],
+    geometry: {
+      type: {
+        type: String,
+        enum: ["Point"],
+      },
+      coordinates: {
+        type: [Number, Number],
+      },
+    },
+    price: Number,
+    description: String,
+    location: String,
+    author: {
+      type: Schema.Types.ObjectId,
+      ref: "User",
+    },
+    reviews: [
+      {
+        type: Schema.Types.ObjectId,
+        ref: "Review",
+      },
+    ],
+  },
+  opts
+);
+
+const Campground = mongoose.model("Campground", campgroundSchema);
+
+//Placeholder random data for testing purposes
+export async function seedCampgrounds() {
+  await Campground.deleteMany({});
+  Array(10)
```

Comment:
Make the number into a variable so we can change the amount of seeds when needed.